ruby-challenges.rb

```diff
@@ -31,3 +63,21 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+
+def palindrome_method (string)
```

Comment:
Love how you use capitalize. Totally works because there is only one letter capitalize. What else can you do to ensure if a word like `tOoT` was given?